internal/moreos/moreos_func-e_test.go

```diff
@@ -136,3 +180,14 @@ func requireFindProcessError(t *testing.T, proc *os.Process, expectedErr error)
 		return err == expectedErr
 	}, 100*time.Millisecond, 5*time.Millisecond, "timeout waiting for expected error %v", expectedErr)
 }
+
+func requireScannedWaitFor(t *testing.T, s *bufio.Scanner, waitFor string) {
```

Comment:
later I will make a refactoring PR as we use scanner in the e2e for the same reasons, even if it could be documented better. Most notably we also avoid `require.Eventually` as it makes crap error messages when it fails. In other words, we used to use `require.Eventually`, that caused very difficult troubleshooting in CI, then we switched off of it.

Even if we fail to do refactoring for whatever reason, when there is copy/paste or something very similar, it is worth commenting the other function or file this is almost the same as. That way we can know to look at it in worst case. Folks who read those comments can help prevent adding a third similar thing later. It also helps reviewers who may have forgotten the other thing.